_inc/client/at-a-glance/style.scss

```diff
@@ -70,6 +72,13 @@
 	}
 }
 
+/* .jp-at-a-glance__stats-cta-description,
```

Comment:
I'll be cleaning up all the commented out code, just want to do a few more rounds of testing first. Commited this so I could update my branch to bring down some newer style adjustments from master.